ksql-engine/src/main/java/io/confluent/ksql/physical/PhysicalPlanBuilder.java

```diff
@@ -200,7 +200,7 @@ private QueryMetadata buildPlanForBareQuery(
 
 
   private QueryMetadata buildPlanForStructuredOutputNode(
-      final String sqlExpression, final SchemaKStream schemaKStream,
+      final String sqlExpression, final SchemaKStream<?> schemaKStream,
```

Comment:
nit: i know i'm picking on stuff that was already here, but... newline for param when split across multiple lines...

Reply:
Agreed, but trying to minimise changes... its a big PR already